docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -4,79 +4,40 @@ lang: en
-2. Add the parameter `customOrgMetadataTemplates` and set its value to the templates folder in your cloned repo. For example:
-```
-{
-  "customOrgMetadataTemplates": "https://github.com/mygitrepo/salesforcedx-templates/tree/main/packages/templates/src/templates"
-}
-```
-
-or the location of a local copy:
+ You can store template files in a local directory, or in a github project. Set ``customOrgMetadataTemplates`` to your custom template location:
+1. Run ``sfdx config:set customOrgMetadataTemplates=<github repo or local template directory>`` command from the terminal inside VS Code. 
+2. Run ``sfdx config:list`` to confirm that the configuration setting is updated. 
 
-```
-{
-  "customOrgMetadataTemplates": "/Users/mydrive/github/devtools/salesforcedx-templates/packages/templates/src/templates"
-}
-```
+**Note:** The `sfdx-config.json` config file in your `<project-folder>/.sfdx` folder now has a new entry in the format,`` "customOrgMetadataTemplates": "<github repo or local template directory>"``.
 
-**Note:** You can also use the CLI to set this parameter. See [CLI Runtime Configuration Values](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_config_values.htm) for more information.
+ See [CLI Runtime Configuration Values](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_config_values.htm) for more information about configuration settings.
 
-## Create a Apex Class with Custom Code
-1. Clone the sample GitHub repo.
-2. Delete all folders except the `apexclass` folder. Also delete all the files except the `DefaultApexClass.cls` file from the `apexclass` folder.
-3. Edit the `DefaultApexClass.cls` file with your custom code:
+## Set Template Location Examples
+-  ``sfdx config:set customOrgMetadataTemplates=https://github.com/vscodeuser/salesforcedx-templates/tree/main/src/templates/ ``sets the ``customOrgMetadataTemplates`` configuration value to a directory on user ``vscodeuser``'s github repo.
+-   ``sfdx config:set customOrgMetadataTemplates=/Users/vscodeuser/CustomTemplateProject/MyCustomTemplates`` sets the ``customOrgMetadataTemplates`` configuration value to the ``MyCustomTemplates`` directory on a local machine.
 
-```
-public with sharing class <%= apiName %> {
-    		public <%= apiName %>(String prop) {
-			this.prop = prop;
-    		}
+## Use Custom Templates On GitHub
+1. Clone this [git repo subdirectory](https://github.com/forcedotcom/salesforcedx-templates/tree/main/src/templates).
+2. In your cloned repo, delete all folders except the folders that contains the templates you want to use. 
+3. Make updates to the custom templates in your repo.
+4. Check that ``customOrgMetadataTemplates`` points to this repo.
 
-		@AuraEnabled public String prop { get;set; }
-	}
-``` 
-**Note:** If you edit the file locally, remember to push changes to your repo. You can also choose to make changes directly in your repo. 
+## Use Local Custom Templates
```

Comment:
Am I understanding correctly that you have to exactly mirror the folder names and file names in the sample repo. I would move that info from note down here and make that explicit in the intro to this section